src/nurse/management/commands/send_notifications.py

```diff
@@ -0,0 +1,14 @@
+from django.core.management.base import BaseCommand
+
+from ._notifications import send_notification
+
+
+class Command(BaseCommand):
+    help = "Sends notifications"
+
+    def handle(self, *args, **options):
+        try:
+            send_notification()
+            self.stdout.write(self.style.SUCCESS("Notifications sent successfully"))
+        except Exception as e:
+            self.stderr.write(self.style.ERROR(f"An error occurred: {str(e)}"))
```

Comment:
We probably want to log that to Sentry as well.
Since it's a management command, I'm wondering if Sentry would catch it if we were to let it uncaught

Reply:
I remove it